backend/src/main/scala/com/softwaremill/bootzooka/Main.scala

```diff
@@ -1,17 +1,15 @@
 package com.softwaremill.bootzooka
 
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
 import com.softwaremill.bootzooka.config.Config
-import com.softwaremill.bootzooka.infrastructure.CorrelationId
 import com.softwaremill.bootzooka.metrics.Metrics
 import com.typesafe.scalalogging.StrictLogging
 import doobie.util.transactor
-import monix.eval.Task
-import monix.execution.Scheduler.Implicits.global
 import sttp.client3.SttpBackend
 
 object Main extends StrictLogging {
   def main(args: Array[String]): Unit = {
-    CorrelationId.init()
```

Comment:
we need to migrate correlation id support as well:) probably implementing https://github.com/softwaremill/monix-correlation-id using fiber-locals inside bootzooka for now, we might extract it as a separate repository later